og_context/og_context.module

```diff
@@ -177,10 +177,29 @@ function og_context_og_invalidate_cache($gids = array()) {
   }
 }
 
+/**
+ * Determines if OG context has been already called.
+ * og_context function should not be called again until it is still running.
+ *
+ * @param bool $called
+ *  Determines if og_context function is still running.
```

Comment:
`Determines if og_context() function has already been executed. Defaults to FALSE.`